SEC/Equities/EquityInstruments.rdf

```diff
@@ -802,12 +901,20 @@
 		<fibo-fnd-utl-av:synonym>has announcement date</fibo-fnd-utl-av:synonym>
 	</owl:ObjectProperty>
 	
-	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasDividendRate">
-		<rdfs:subPropertyOf rdf:resource="&lcc-lr;has"/>
-		<rdfs:label>has dividend rate</rdfs:label>
-		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;PreferredShareCoupon"/>
-		<rdfs:range rdf:resource="&fibo-fnd-acc-cur;PercentageMonetaryAmount"/>
-		<skos:definition>indicates a pre-arranged dividend rate</skos:definition>
+	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasDividendGracePeriod">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasDuration"/>
+		<rdfs:label>has dividend grace period</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;PreferredDividend"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;Duration"/>
+		<skos:definition>indicates a period of time after a dividend payment becomes due, before the issuer is subject to penalties</skos:definition>
+	</owl:ObjectProperty>
+	
+	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasEarliestRedemptionDate">
+		<rdfs:subPropertyOf rdf:resource="&fibo-fnd-dt-fd;hasExplicitDate"/>
+		<rdfs:label>has earliest redemption date</rdfs:label>
+		<rdfs:domain rdf:resource="&fibo-sec-eq-eq;EquityRedemptionProvision"/>
+		<rdfs:range rdf:resource="&fibo-fnd-dt-fd;ExplicitDate"/>
```

Comment:
Surely should be able to express this as a duration after effective date

Reply:
No - most contracts that have earliest redemption dates specify a precise date.  If you have a reference for your comment it would be great to get the counter example - the FCT did not find one, and this corresponds to what Golden Source and Citi have in their models.

Reply:
OK